### ISSUE TITLE
'Update Grav Now' notification causes accidental update of Grav

### ISSUE THREAD

**Issue by redrohX** [getgrav/grav-plugin-admin#1257]
The way the 'Update Grav Now' notification is implemented makes it really easy to accidentally click on the button. Obviously updating is important, but not when you're in the middle of doing something else. 

Consider this scenario: 
- as an admin user I open a page to edit
- I want to select a different language from the dropdown
- The moment I want to click on the dropdown the update notification pops up and I accidentally click on the 'Update Grav Now' button

I think as an improvement there are several options to consider:
- make sure the notification is less intrusive
  - doesn't pop-up after the rest of the content is loaded
  - Give it a fixed position so content below isn't forced down when it finally loads
  - put the notification in a different location
- Ask for a confirmation after clicking on the 'Update Grav Now' button.

![schermafbeelding 2017-11-03 om 13 29 39](https://user-images.githubusercontent.com/1741055/32374114-a4a55e48-c09c-11e7-84b3-b2b52dc40dd4.png) 
_Try not to click the language dropdown before the notification is visible_

**Comment by rhukster** [comment]
I'll discuss this with the team...

**Comment by w00fz** [comment]
This is handled much better now. 

First I made sure to add a confirmation modal, so you won't accidentally update by clicking on the button once.
Additionally, I have also added a cool down counter, 5 seconds. This means the button won't be enabled at all for at least 5 seconds after it appears. This way even clicking on it won't even trigger the confirmation modal for 5 seconds, alleviating the issue of accidentally triggering an update when the button appears under your cursor.


Sorry this took so long, it will be in 1.10 now though.
